models/zone.model.ts

```diff
@@ -0,0 +1,26 @@
+import { SterilizedCat, UnsterilizedCat } from './cat.model'
+
+enum Status {
+    todo = 'To Do',
```

Comment:
you can use this enum, defined and exported only here, and apply the changes requested above, in the zone schema. Also the enum declaration in zone schema can be removed